### PR TITLE
Add instruction for first build

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
I always forget the command and like that, you just copy-paste and here you go!

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
cc @Hoverbear

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ For Windows, you also need to install following software:
 ## Build
 
 ```
+$ git submodule update --init --recursive # if you just cloned the repository
```
Comment (BusyJay) [inline README.md]: Or use the script `scripts/reset-submodule`, which only update the required submodules. Although it may not work on Windows.

Comment (GuillaumeGomez) [reply README.md]: So maybe not a great idea to propose it by default. ;)

**Comment by siddontang** [comment]
LGTM

**Comment by BusyJay** [comment]
Thank you!

**Pull request merged**